src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/CompletableHelper.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services;
```

Comment:
object name says `CompletableHelper` and only has some abstract runnable handling. does suggest to perhaps separate out to, let's say, `helper` package and not `services`, does it not?